### PR TITLE
SWD-123: A new feature

### PR DESCRIPTION
**Pull request by sotosant**
### Summary

Adding caption

### Reviewer

@rockykeele please review

### CODE REVIEW

**Review by githubteacher — CHANGES_REQUESTED** [review]
@sotosant, what if we removed the `Caption here` text from your post

File: _posts/0000-01-09-sotosant.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Miguel
```
Comment (githubteacher) [inline _posts/0000-01-09-sotosant.md:6]: I like this 👍.

**Review by githubteacher — APPROVED** [review]
**Let's :shipit: !**

**Comment by sotosant** [comment]
done

**Pull request merged**